bin/manual_testing

```diff
@@ -3,16 +3,32 @@
 set -e
 
 if [ -z $1 ]; then
-    echo "Usage: $0 plugin"
+    echo "Usage: $0 plugin [version]"
```

Comment:
I think it would be a good idea to add information about the possibility to install .zips here (and to README usage section).